src/js/themes/base.d.ts

```diff
@@ -278,6 +278,9 @@ export interface ThemeType {
         duration?: string;
       };
     };
+    backgrounds?: {
```

Comment:
This is feeling just a bit unaligned with how the `background` property works for components, where a color can be a background. Can I put a color HEX value here to add additional background options or is this just for images and gradients?

Reply:
The current implementation is definitely geared towards images and gradients. I definitely see your point about the misalignment for how background property works. I agree that if a string is provided, we should be able to support all flavors of that string.

I will rework.

Reply:
Update. Made decent progress. Supporting string, dark-light object, and BackgroundType object, where string may be a hex value, color name from theme, url, or linear-gradient.

Need to work through one edge case where `background: { ..., dark: true, ... }` is not accounted for.

![Screen Shot 2022-09-02 at 11 11 19 AM](https://user-images.githubusercontent.com/1756948/188205156-ac281ecd-9830-41c8-b589-cb2cead3e4c4.png)

Reply:
All cases should now be accounted for.